<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitService</class>
 <widget class="QDialog" name="InitService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>1</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="margin">
             <number>4</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>服务流程</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <property name="margin">
                <number>4</number>
               </property>
               <item>
                <widget class="QListWidget" name="serviceList"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1,1">
            <property name="margin">
             <number>2</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_9" native="true">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="margin">
                <number>1</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>名   称:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="serviceName">
                 <property name="minimumSize">
                  <size>
                   <width>150</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>222</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="addService">
                 <property name="minimumSize">
                  <size>
                   <width>55</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>添  加</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="removeService">
                 <property name="minimumSize">
                  <size>
                   <width>55</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>移  除</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="applyUpdate">
                 <property name="minimumSize">
                  <size>
                   <width>55</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>保存修改</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="saveToDisk">
                 <property name="minimumSize">
                  <size>
                   <width>55</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>保存文件</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="loadFromDisk">
                 <property name="minimumSize">
                  <size>
                   <width>55</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>本地加载</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>输入参数</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="margin">
                <number>4</number>
               </property>
               <item>
                <widget class="QWidget" name="inputWidget" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>输出结果</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="margin">
                <number>4</number>
               </property>
               <item>
                <widget class="QWidget" name="outputWidget" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
